chore(docs): update release preview for Spring Boot 2.7.18 (#434)
spinnaker/spinnaker.io: content/en/docs/releases/next-release-preview/index.md

[orca: front50.useTriggeredByEndpoint](https://spinnaker.io/changelogs/1.31.0-changelog/#orca)

Both of these features are now enabled by default.

### Spring Boot 2.7.18

As part of the continued effort to upgrade Spring Boot, Spinnaker 1.35.0 now uses Spring Boot 2.7.18, an upgrade from Spinnaker 1.34.0`s use of Spring Boot 2.6.15. Spring Boot 2.7 considers session data cached by Spring Boot 2.6 invalid.  Therefore, users with cached sessions will be unable to log in until the invalid information is removed from the cache. Open browser windows to Spinnaker are unresponsive after the deployment until they’re reloaded. 
Executing:

    $ redis-cli keys "spring:session*" | xargs redis-cli del

on Gate's redis instance removes the cached session information.

> Spring Boot 2.7 bring given below major changes in spinnaker:

Reviewer: ```suggestion
Spring Boot 2.7 brings with it the following changes:
```
